tests/ga/test_exthandlers_exthandlerinstance.py

```diff
@@ -29,6 +29,7 @@ def setUp(self):
 
     def tearDown(self):
         self.mock_get_base_dir.stop()
+        super(ExtHandlerInstanceTestCase, self).tearDown()
```

Comment:
There were a couple of these where a subclass overrides the base class (AgentTestCase) and don't call into it. I added the call.